File: core-java-practice/gcr-codebase/programming-elements/CelsiusToFahrenheitConversion.java
package Dayone;

import java.util.Scanner;

public class CelsiusToFahrenheitConversion {
  public static void main(String [] args){
    //Write a program that takes temprature in Celsius as input and converts it to fahrenheit using the formula Fahrenheit =(Celsius*9/5)+32
    Scanner sc=new Scanner (System.in);
    double Celsius=sc.nextDouble();
    double Fahrenheit =(Celsius*9/5)+32;
    System.out.println(" Fahrenheit: "+  Fahrenheit);
  }
}

File: core-csharp-practice/gcr-codebase/programming-elements/CelsiusToFahrenheitConversion.cs
using System;
namespace ProgrammingElements{
    class CelsiusToFahrenheitConversion{
        static void Main(string[] args){
            // Write a program that takes temperature in Celsius as input
            // and converts it to Fahrenheit
            double celsius = double.Parse(Console.ReadLine());
            double fahrenheit = (celsius * 9 / 5) + 32;
            onsole.WriteLine("Fahrenheit: " + fahrenheit);
        }
    }
}
